docs/source/release/v6.9.0/index.rst

```diff
@@ -16,7 +16,14 @@ Mantid 6.9.0 Release Notes
 
```

Comment:
Should we remove the section above this line? The one about:
```
.. warning:: This release is still under construction. ...
```

Reply:
Oops I missed that, thanks!